### PR TITLE
Adding Help IDs and renaming dialog

### PR DESCRIPTION
**Pull request by AlexSananka**
@dannyparsons here I added the Help Ids that were not added and renamed water balance to end of rains.

### CODE REVIEW

**Comment by shadrackkibet** [comment]
reviewing this now.

File: instat/dlgWaterBalance.Designer.vb [hunk 1]
```diff
@@ -292,8 +300,8 @@ Partial Class dlgWaterBalance
         Me.MinimizeBox = False
         Me.Name = "dlgWaterBalance"
         Me.StartPosition = System.Windows.Forms.FormStartPosition.CenterScreen
-        Me.Tag = "Water_Balance"
-        Me.Text = "Water Balance"
+        Me.Tag = ""
+        Me.Text = "End of rains"
```
Comment (shadrackkibet) [inline instat/dlgWaterBalance.Designer.vb]: @dannyparsons  should this be "End of Rains" or "End of rains".?

Comment (dannyparsons) [inline instat/dlgWaterBalance.Designer.vb]: End of Rains

**Comment by AlexSananka** [comment]
@dannyparsons This is ready.

**Pull request merged**